js_modules/dagster-ui/packages/ui-core/src/assets/usePaginatedAssetEvents.tsx

```diff
@@ -70,42 +64,38 @@ export function usePaginatedAssetEvents(
           cursor,
           before,
           after: afterParam,
-          eventTypeSelector: params.status ?? MaterializationHistoryEventTypeSelector.ALL,
+          eventTypeSelectors: params.statuses ?? [
+            AssetEventHistoryEventTypeSelector.MATERIALIZATION,
+            AssetEventHistoryEventTypeSelector.OBSERVATION,
+            AssetEventHistoryEventTypeSelector.FAILED_TO_MATERIALIZE,
+          ],
         },
       });
       setLoading(false);
 
       const asset = data?.assetOrError.__typename === 'Asset' ? data?.assetOrError : null;
 
-      const {materializations, observations} = clipEventsToSharedMinimumTime(
-        asset?.assetMaterializationHistory?.results || [],
-        asset?.assetObservations || [],
-        100,
-      );
-
       setLoaded(true);
-      setMaterializations((loaded) =>
-        uniqBy([...loaded, ...materializations], (e) => `${e.runId}${e.timestamp}`),
+      setEvents((loaded) =>
```

Comment:
I was very unsure of what i was doing here

Reply:
I am not 100% sure what this logic was doing, but now that the two arrays are merged, the uniq will remove one of the events if both a materialization and an observation came from the same run (which is sort of a valid thing to do)

Maybe we throw the e.__typename in the uniq key on line 82 to make the behavior the same as it was before?